config.c

```diff
@@ -51,21 +70,25 @@ void init_default_config(struct mako_config *config) {
 	config->hidden_style.format = strdup("(%h more)");
 	config->hidden_style.spec.format = true;
 
-	config->output = strdup("");
-	config->layer = ZWLR_LAYER_SHELL_V1_LAYER_TOP;
+	new_surface->name = strdup("(root)");
 
-	config->max_visible = 5;
-	config->max_history = 5;
+	new_surface->max_visible = 5;
+	new_surface->max_history = 5;
 	config->sort_criteria = MAKO_SORT_CRITERIA_TIME;
 	config->sort_asc = 0;
 
 	config->button_bindings.left = MAKO_BINDING_INVOKE_DEFAULT_ACTION;
 	config->button_bindings.right = MAKO_BINDING_DISMISS;
 	config->button_bindings.middle = MAKO_BINDING_NONE;
 	config->touch = MAKO_BINDING_DISMISS;
+}
 
-	config->anchor =
-		ZWLR_LAYER_SURFACE_V1_ANCHOR_TOP | ZWLR_LAYER_SURFACE_V1_ANCHOR_RIGHT;
+static void free_surface_config(struct mako_surface_config *surface) {
```

Comment:
Style: this should use the `destroy_` prefix in line with other functions which free the given struct.